src/cr-image/grayscale_image.cr

```diff
@@ -71,6 +71,13 @@ class CrImage::GrayscaleImage < CrImage::Image
     nil
   end
 
+  # Run provided block with the `ChannelType::Gray` channels and channel types.
+  def each_channel(& : (Array(UInt8), ChannelType) -> Nil) : Nil
+    yield @gray, ChannelType::Gray
+    yield @alpha, ChannelType::Alpha
+    nil
+  end
```

Comment:
Fix bug where resizing methods (crop, resize, etc.) wouldn't resize the alpha channel due to using the `each_color_channel`.